groovy/gradle.persistence/manifest.mf

```diff
@@ -0,0 +1,5 @@
+Manifest-Version: 1.0
+AutoUpdate-Show-In-Client: true
```

Comment:
The module is called "bridge" shouldn't it automatically activate when necessary modules are present? That would imply this module should be `eager`.

Reply:
Lack of knowledge again. Thanks!